orc8r/tools/fab/dev_utils.py

```diff
@@ -192,6 +198,70 @@ def get_hardware_id_from_vagrant(vm_name: str) -> str:
     return str(hardware_id)
 
 
+def get_hardware_id_from_docker(
+    location_docker_compose: str =
+    '../../../lte/gateway/python/integ_tests/federated_tests/docker/',
+) -> str:
+    """
+    Get the magmad hardware ID from magma running on docker from Vagrant
+    Args:
+        location_docker_compose: location of docker compose used to run FEG
+        by default lte/gateway/python/integ_tests/federated_tests/docker
+    Returns:
+        Hardware snowflake from the VM
+    """
+    with lcd('docker'), hide('output', 'running', 'warnings'), \
+            cd(location_docker_compose):
+        hardware_id = local(
+            'docker-compose exec magmad bash -c "cat /etc/snowflake"',
+        capture=True,
+        )
+    return str(hardware_id)
+
+
+def delete_agw_certs(vm_name: str):
```

Comment:
These delete_* functions can probably be run on the host -- they're deleting files that exist on the host, are owned/used by the agw, and only happen to be created (for ease of dev bootstrapping) by the orc8r

Reply:
Certs on gateways are stored on `/var/opt/magma/certs` 
That folder is only visible to Vagrant VM
The only shared folder is `/home/vagrant/magma` which is your local magma folder

Reply:
Yeah I misread the purpose of these fns -- I thought it was only for running in orc8r. Yes it makes total sense to do it this way given that these would be run from a gw